src/main/java/com/solr/clientwrapper/infrastructure/entity/Employee.java

```diff
@@ -0,0 +1,26 @@
+package com.solr.clientwrapper.infrastructure.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.apache.solr.client.solrj.beans.Field;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.solr.core.mapping.SolrDocument;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@SolrDocument(collection = "employee")
```

Comment:
Why is this "employee" collection hardcoded...

Reply:
that class created by you

Reply:
I think you have created your branch from the older main branch. Please pull the new main branch and merge the main to your branch. (Do not merge your branch to main branch)

Reply:
actually i have created branch new main branch. but i thought that all code will be save on my branch so i have add all old file for reference.